PROTO_tests/tests/series_test.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+import arkouda as ak
+
+
+class TestSeries:
+    def test_series_creation(self):
```

Comment:
`ak.Series` can have pdarray, Strings and Categorical values. I would add a file level global

```python
DTYPES = [ak.int64, ak.uint64, ak.bool, ak.float64, ak.bigint, ak.str_]
```
and then update this to function definition to parameterize on `DTYPES`. Also, add an `if` block for `dtype == ak.str_` to test `Categorical`. 

Then, we should split the error handling testing into its own function.

Reply:
Added `dtype` testing for several functions where applicable. After going through some issues with @pierce314159 , there are some larger issues that can be readdressed later with the handling of `Categorical` for strings on specific tests. It is properly handled in some cases, in those cases it is tested.

Reply:
Is #2717 part of this? If there are issues related to it, then Issues need to be added to GitHub and linked here so that they are tracked and can be addressed if they will not be resolved as part of this PR.